src/index.ts

```diff
@@ -25,6 +25,22 @@ export interface Collection<K, V> extends Map<K, V> {
 export class Collection<K, V> extends Map<K, V> {
 	public static readonly default: typeof Collection = Collection;
 
+	/**
+	 * Obtains an element if the key exists, otherwise sets and returns the value provided by the default value generator.
```

Comment:
Forgot to send this one, open to suggestions though
```suggestion
	 * Obtains the value of the given key if set, otherwise sets the value provided by the default value generator and returns it.
```